GitTfs/Commands/Init.cs

```diff
@@ -102,6 +102,13 @@ private void DoGitInitDb()
             }
             globals.Repository = gitHelper.MakeRepository(globals.GitDir);
 
+            if (string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.name"))
+                && string.IsNullOrWhiteSpace(globals.Repository.GetConfig("user.email")))
+            {
+                globals.Repository.SetConfig("user.name", "git-tfs");
+                globals.Repository.SetConfig("user.email", "placeholder@git-tfs.com");
+            }
```

Comment:
I found [some docs](https://confluence.jetbrains.com/display/TCD4/6.Properties+and+environment+variables), but the settings page it refers to (Properties and environment variables) doesn't seem to exist. I looked at the appveyor CI build, and see that it's running the unit tests. Is there a reason to keep both around?